src/lib/service-runner-schema.json

```diff
@@ -168,6 +168,9 @@
         },
         "name": {
           "type": "string"
+        },
+        "description": {
+          "type": "string"
```

Comment:
```suggestion
          "type": "string",
          "description": "A verbose description of the application. GitHub Flavored Markdown syntax MAY be used for rich text representation."
```

Reply:
![image](https://user-images.githubusercontent.com/364566/66683119-8a53b000-ec2b-11e9-9f04-5de4fe3c48e1.png)

Reply:
Alot of offline convo and we settled on a different but similar solution in the PR update